src/groups/special_orthogonal_group.jl

```diff
@@ -103,6 +103,9 @@ ManifoldsBase.log(::SpecialOrthogonalGroup{ManifoldsBase.TypeParameter{Tuple{4}}
 @doc "$(_doc_log_O4_id)"
 ManifoldsBase.log!(::SpecialOrthogonalGroup{ManifoldsBase.TypeParameter{Tuple{4}}}, X, g)
 
+pull_back_tangent!(::SpecialOrthogonalGroup, Y, g, X; kwargs...) = (Y .= X)
+push_forward_tangent!(::SpecialOrthogonalGroup, Y, g, X; kwargs...) = (Y .= X)
```

Comment:
```suggestion
pull_back_tangent!(::SpecialOrthogonalGroup, Y, g, X; kwargs...) = copyto!(Y, X)
push_forward_tangent!(::SpecialOrthogonalGroup, Y, g, X; kwargs...) = copyto!(Y, X)
```
This should be easier to handle for the compiler.

Reply:
I always chose the wrong one of these two. consistently. ;)